pkg/reconciler/revision/resources/deploy.go

```diff
@@ -237,15 +232,15 @@ func MakeDeployment(rev *v1.Revision,
 		replicaCount, _ = strconv.Atoi(ann)
 	}
 
+	labels := makeLabels(rev)
+	anns := makeAnnotations(rev)
```

Comment:
Looks like we exclude more labels and annotations than before. Should we add tests to cover?

Reply:
We have added TestMakeAnnotations to cover the new method. It excludes more annotations, but those are more GarbageCollector related ones (same as the already filtered lastPinned) which have no relevance outside of Revision.